tests/sibernetic/README.md

```diff
@@ -0,0 +1,20 @@
+## Test conversion of Sibernetic output to WCON
+
+Example output produced by Sibernetic & some Python scripts for converting this to WCON.
+
+**Work in progress!**
+
+Sibernetic original data is in [crawling_at_agar.txt](crawling_at_agar.txt).
+
+The file [generate_wcon.py](generate_wcon.py) can be used to convert this into WCON.
+
+Example of the output in WCON can be seen [here](sibernetic_test_small.wcon) (or a longer version [here](sibernetic_test_full.wcon)).
+
+The activity of the worm (generated from the original data, not WCON) can be seen below:
+
+![outline](outline.png)
+
+The speed of the worm during this can be plotted:
+
+![speed](speed.png)
```

Comment:
Same here.  (And same with the generated WCON files.)  The only thing would be if you wanted to keep something as part of a test that could automatically determine whether the conversion produced the expected results.